bats/tests/helpers/vm.bash

```diff
@@ -39,6 +39,10 @@ factory_reset() {
     if [ "$BATS_TEST_NUMBER" -gt 1 ]; then
         capture_logs
     fi
+    ENGINE_CONFIG="${PATH_APP_HOME}/rd-engine.json"
+    if [ -f "$ENGINE_CONFIG" ] && [ -z "$(cat "$ENGINE_CONFIG")" ]; then
+        rm "$ENGINE_CONFIG"
+    fi
```

Comment:
This can be simplified to
```suggestion
    if [[ -f $ENGINE_CONFIG && ! -s $ENGINE_CONFIG ]]; then
        rm "$ENGINE_CONFIG"
    fi
```

Or even simpler

```suggestion
    # Delete empty config file
    if [[ ! -s $ENGINE_CONFIG ]]; then
        rm -f "$ENGINE_CONFIG"
    fi
```

But maybe a comment about how/why we can ever end up with an empty config file is more important: Isn't that a bug in the app that should never happen?

Reply:
I needed this, but I didn't determine why, so I'll just delete the code and figure it out. Meanwhile having a gained a bit of an understanding about `[[ ... ]]`